apps/frontend/src/components/proposal/TableActionsDropdownMenu.tsx

```diff
@@ -112,6 +118,18 @@ const TableActionsDropdownMenu = ({
     null
   );
   const [open, setOpen] = React.useState<boolean>(false);
+  const [menuItems, setMenuItems] = React.useState(getMenuItems());
+  const [searchParams] = useSearchParams();
+  const { search } = useLocation();
```

Comment:
Do you think this is necessary?

Reply:
Yes. The reason is, 

When there is a change in the query params caused by different components, the current component doesnt know about it. The `searchParams` from `useSearchParams` does not reinstantiate and hence, we can't use it in `useEffect`, but the `search` from `useLocation` does side effect. 

https://reactrouter.com/en/main/hooks/use-location

@janosbabik